tests/tape/tapes/test_qnode.py

```diff
@@ -600,3 +601,40 @@ def func(x, y):
 
         assert np.allclose(res, res2, atol=tol, rtol=0)
         assert func.qtape is not old_tape
+
+
+class TestQNodeCollection:
+    """Unittests for the QNodeCollection"""
+
+    @pytest.fixture
+    def enable_tape_mode(self):
+        qml.enable_tape()
+        yield
+        qml.disable_tape()
+
+    def test_multi_thread(self, enable_tape_mode):

```

Comment:
It is useful to add test docstrings, so that future developers can straight away see the intention of the test :)
```suggestion
    def test_multi_thread(self, enable_tape_mode):
        """Test that multi-threaded queuing in tape mode works correctly"""
```